plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduModule.java

```diff
@@ -83,20 +104,47 @@ Procedure getDropRangePartitionProcedure(RangePartitionProcedures procedures)
         return procedures.getDropPartitionProcedure();
     }
 
-    @Singleton
-    @Provides
-    KuduClientSession createKuduClientSession(KuduClientConfig config)
+    private static class NoneAuthenticationModule
+            extends AbstractConfigurationAwareModule {
+        @Override
+        public void setup(Binder binder)
+        {
+        }
+
+        @Provides
+        @Singleton
+        public static KuduClientSession createKuduClientSession(KuduClientConfig config)
+        {
+            return KuduModule.createKuduClientSession(config, KuduClient.KuduClientBuilder::build);
+        }
+    }
+
+    private static class KerberosAuthenticationModule
+            extends AbstractConfigurationAwareModule {
+        @Override
+        public void setup(Binder binder) {
```

Comment:
Can we make it compatible with the existing code style ?

Reply:
I'm unsure what you mean, but I fixed the checkstyle violations if that is what you meant.

Reply:
Split it out into a separate module, let me know if that is not what you meant.